core/Columns/Dimension.php

```diff
@@ -417,7 +419,12 @@ public function groupValue($value, $idSite)
             case Dimension::TYPE_BOOL:
                 return !empty($value) ? '1' : '0';
             case Dimension::TYPE_DURATION_MS:
-                return number_format($value / 1000, 2) * 1000; // because we divide we need to group them and cannot do this in formatting step
+                // Log a warning if the value isn't actually numeric
+                if (!is_numeric($value)) {
+                    $message = Piwik::translate('General_ValidatorErrorNotANumber') . ' ' . $value;
+                    StaticContainer::get(LoggerInterface::class)->warning($message, ['value' => $value, 'idSite' => $idSite]);
+                }
+                return number_format(floatval($value) / 1000, 2) * 1000; // because we divide we need to group them and cannot do this in formatting step
```

Comment:
@snake14 This will still fail,  the error is due to formatted value being multiplied
Eg : `$value = "8,883.86";echo $value * 1000;`

Reply:
Interesting. I wonder why that hasn't been an issue sooner then. I guess I can cast that instead.

Reply:
Thank you. I decided to cast both the value and formatted value just to be safe.